doc/12-icinga2-api.md

```diff
@@ -591,13 +591,13 @@ URL path when querying a single object. For objects with composite names
 (e.g. services) the full name (e.g. `example.localdomain!http`) must be specified:
 
 ```
-$ curl -k -s -u root:icinga 'https://localhost:5665/v1/objects/services/example.localdomain!http'
+$ curl -k -s -S -i -u root:icinga 'https://localhost:5665/v1/objects/services/example.localdomain!http'
 ```
 
 You can limit the output to specific attributes using the `attrs` URL parameter:
 
 ```
-$ curl -k -s -u root:icinga 'https://localhost:5665/v1/objects/hosts/example.localdomain?attrs=name&attrs=address&pretty=1'
+$ curl -k -s -S -i -u root:icinga 'https://localhost:5665/v1/objects/hosts/example.localdomain?attrs=name&attrs=address&pretty=1'
```

Comment:
With the -i option curl also outputs headers like this:

```
$ curl -ksSiu root:18d403d49f9443b0 'https://[::1]:5665/v1'
HTTP/1.1 200 OK
Server: Icinga/v2.12.0-rc1-52-g04704a49a
Content-Type: text/html
Content-Length: 367

<html><head><title>Icinga 2</title></head><h1>Hello from Icinga 2 (Version: v2.12.0-rc1-52-g04704a49a)!</h1><p>You are authenticated as <b>root</b>. Your user has the following permissions:</p> <ul><li>*</li></ul><p>More information about API requests is available in the <a href="https://icinga.com/docs/icinga2/latest/" target="_blank">documentation</a>.</p></html>
```

But with your changes -i seems not to do that as the result JSON directly follows the curl command (no headers between):

    ```
    curl ...
    {"status":"OK}
    ```

Resolve this confusion like this:

    ```
    curl ...
    ```

    ```
    {"status":"OK}
    ```

... for all lines you've changes which are not just:

    ```
    curl ...
    ```

Reply:
Looks like you forget to do this in the one place where this comment was added.